sync/class.jetpack-sync-module-users.php

```diff
@@ -195,70 +180,52 @@ public function deleted_user_handler( $deleted_user_id, $reassigned_user_id = ''
 		do_action( 'jetpack_deleted_user', $deleted_user_id, $reassigned_user_id, $is_multisite );
 	}
 
-	public function edited_user_handler( $user_id ) {
-		/**
-		 * Fires when a user is edited on a site
-		 *
-		 * @since 5.4.0
-		 *
-		 * @param int $user_id - ID of the edited user
-		 */
-		do_action( 'jetpack_user_edited', $user_id );
-	}
-
 	function user_register_handler( $user_id, $old_user_data = null ) {
 		// ensure we only sync users who are members of the current blog
 		if ( ! is_user_member_of_blog( $user_id, get_current_blog_id() ) ) {
 			return;
 		}
-		$raw_user = get_user_by( 'id', $user_id );
-		$user = $this->sanitize_user( $raw_user );
-
-		if ( 'user_register' === current_filter() ) {
-			/**
-			 * Fires when a new user is registered on a site
-			 *
-			 * @since 4.9.0
-			 *
-			 * @param object The WP_User object
-			 */
-			do_action( 'jetpack_sync_register_user', $user );
-
-			return;
-		}
+		/**
+		 * Fires when a new user is registered on a site
+		 *
+		 * @since 4.9.0
```

Comment:
Thank you for adding documentation! Could you also add the `@module sync` text to you Sync hooks, so it gets categorized by the parser?
https://developer.jetpack.com/module/sync/

Thank you!

Reply:
noted, do you have time for a quick PR to add the missing ones? 😊

Reply:
Not right now, no, unfortunately, but I'll keep it in mind :)